ui/client/components/graph/GraphPartialsInTS/createPaper.ts

```diff
@@ -57,7 +58,9 @@ function getPaper(opts: dia.Paper.Options, canWrite: boolean) {
       radius: 60,
     },
   }
-
+  //we pass it via attributes to drawGraph so that same arrowMarker id is used
+  paper.attributes = {}
+  paper.attributes["arrowMarker"] = uniqueArrowMarker
```

Comment:
can we be sure there are no other attributes? Can we just write paper.attributes = { "arrowMarker": uniqueArrowMarker } ?

Reply:
"Please use set to update the attributes instead of modifying them directly." https://backbonejs.org/#Model-attributes